src/Data.js

```diff
@@ -211,6 +211,8 @@ CM.ConfigData.ScaleCutoff = {type: 'numscale', group: 'Notation', label: 'Notati
 CM.ConfigData.GCTimer = {type: 'bool', group: 'Miscellaneous', label: ['Golden Cookie Timer OFF', 'Golden Cookie Timer ON'], desc: 'A timer on the Golden Cookie when it has been spawned', toggle: true, func: function() {CM.Disp.ToggleGCTimer();}};
 CM.ConfigData.Favicon = {type: 'bool', group: 'Miscellaneous', label: ['Favicon OFF', 'Favicon ON'], desc: 'Update favicon with Golden/Wrath Cookie', toggle: true, func: function() {CM.Disp.UpdateFavicon();}};
 CM.ConfigData.WrinklerButtons = {type: 'bool', group: 'Miscellaneous', label: ['Extra Buttons OFF', 'Extra Buttons ON'], desc: 'Show buttons for popping wrinklers at bottom of cookie section', toggle: true, func: function() {CM.Disp.UpdateWrinklerButtons();}};
+CM.ConfigData.ShowMysteriousAchievements = {type: 'bool', group: 'Miscellaneous', label: ['Show Mysterious Achievements OFF', 'Show Mysterious Achievements ON'], desc: 'Show mysterious achievemennts in stats', toggle: true, func: function() {}};
```

Comment:
The group of this option should be "statistics". You can put it directly under `CM.ConfigData.MissingUpgrades`. Could you also change the name to `ShowMissingAchievements`, for consistency's sake :)

Edit: I just thought of something. Depending on the final implementation of this function you will likely also need to add the function that shows or hides the achievements (for now this is `CM.Disp.ReplaceCrateTooltipAchievements`) as the function here. See `CM.ConfigData.Favicon` for an example. This will make it so that when a user toggles the setting the achievements will also update their display. However, I can also see implementations where this is not necessary!

Reply:
CM.Disp.ReplaceCrateTooltipAchievements is fired from the initialization and the game will call the replaced functions after certain seconds and inside the call it will check for the variables of the options. Therefore, it cannot update it immediately, but after few seconds when the gam calls the function and reads the options